### DISCUSSION TITLE
Transitions for x-text values?

### DISCUSSION THREAD

**Discussion by Biszu** [alpinejs/alpine#4212] [1. Help]
Hey 👋

Is there a way to add an animation/transition to x-text updates?
I'm trying to achieve something similar to this: https://jsfiddle.net/adrianopolis/ezzLgLhy/
Of course without changing the color or any other properties of the element, just updating the text content.

I couldn't find any similar topics or figure it out myself, so I'd appreciate all suggestions!

**Comment by ekwoka** [comment]
You just have two elements and transition between those.
